frappe/custom/doctype/customize_form/customize_form.py

```diff
@@ -794,6 +794,7 @@ def get_link_filters_from_doc_without_customisations(doctype, fieldname):
 	"is_virtual": "Check",
 	"link_filters": "JSON",
 	"placeholder": "Data",
+	"allow_others": "Check",
```

Comment:
Nope. This should never be customisable. It's fine in custom field but if you allow modifying standard field them all validation is pointless. Business logic also can't assume that this field will contain known values.